src/SIL.XForge.Scripture/ClientApp/src/app/translate/editor/editor.component.ts

```diff
@@ -1637,11 +1637,13 @@ export class EditorComponent extends DataLoadingComponent implements OnDestroy,
   }
 
   private hasNewContent(thread: NoteThreadDoc): boolean {
-    const noteId: string | undefined = thread.data?.notes[thread.data?.notes.length - 1].dataId;
-    if (noteId == null || this.projectUserConfigDoc?.data == null) {
-      return false;
-    }
-    return !this.projectUserConfigDoc.data.noteRefsRead.includes(noteId);
+    if (thread.data == null || this.projectUserConfigDoc?.data == null) return false;
+    // look for any note that has not been read and was authored by another user
+    const noteRefsRead: string[] = this.projectUserConfigDoc.data.noteRefsRead;
+    const unreadNoteIndex: number = thread.data.notes.findIndex(
+      n => n.ownerRef !== this.userService.currentUserId && !noteRefsRead.includes(n.dataId)
+    );
+    return unreadNoteIndex > -1;
```

Comment:
How about using `some()`:
``` suggestion
    return thread.data.notes.some(
      n => n.ownerRef !== this.userService.currentUserId && !noteRefsRead.includes(n.dataId)
    );
```

Reply:
I like it. I had thought I tried the some() method, but I must not have.